test/runtests.jl

```diff
@@ -38,16 +38,16 @@ else
     @test typeof(getproperties(t)) == Dict{String,Tuple{Any,Bool}}
     @test typeof(getproperties(t,dev*"/ai0")) == Dict{String,Tuple{Any,Bool}}
     @test typeof(t) == NIDAQ.AITask
-    @test start(t) == nothing
+    @test NIDAQ.start(t) == nothing
```

Comment:
qualifying with `NIDAQ` would not be needed if start was still exported.  does that not work for some reason?

Reply:
Makes sense, not sure why I didn't export it. I'll try it and make sure I'm not forgetting some problem with it.

Reply:
It turns out I did export `start`, see [this line](https://github.com/Cody-G/NIDAQ.jl/blob/cjg/julia7/src/NIDAQ.jl#L26).  The reason I qualified the calls to start is to avoid an ambiguity error in Julia 0.7 (`Base.start` still exists in 0.7, though it's deprecated.)

Reply:
if there is not a way to use Compat to avoid the warnings on 0.7 then i just wouldn't worry about it and leave `start` unqualified.  1.0 is not just the future, it's the present.

Reply:
I don't see a way to do this using Compat.  I can just leave it unqualified, and this will cause tests to fail on 0.7.  But the package will still work on 0.7, so I'd rather not bump the minimum version just yet (0.7 still comes in handy for us).  Sound good?